### PR TITLE
Event cap documentation: api and data dictionary

### PR DESCRIPTION
**Pull request by jaredmcdonald**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,6 +282,7 @@ Requires authentication: Yes
 | `id`         | int  |                |
 | `start_date` | int  | Unix timestamp |
 | `end_date`   | int  | Unix timestamp |
+| `is_full`    | bool | Whether the timeslot is full. Only present on the [event list](#list-all-public-events) and [organization event list](#list-organization-events) endpoints; `null` elsewhere. |
```
Comment (kylehg) [inline README.md]: Where is "elsewhere" in this case? Aren't those the only two endpoints that return event objects?

Comment (jaredmcdonald) [reply README.md]: all of the attendances endpoints embed timeslots, and also the responses when you create or edit an event

Comment (kylehg) [reply README.md]: Ah gotcha. Makes sense

**Comment by jaredmcdonald** [comment]
@kylehg quick bump to look at this again, I added the filter option and a line in the data dictionary (was planning to hold off merging until we turn this on in production)

**Comment by kylehg** [comment]
LG

**Pull request merged**